src/gridfs-stream/download.ts

```diff
@@ -1,5 +1,54 @@
+import type { FindOptions } from './../operations/find';
+import type { Document, AnyError } from './../types.d';
+import type { Cursor } from './../cursor/cursor';
 import { Readable } from 'stream';
 import type { Callback } from '../types';
+import type { Collection, ReadPreference } from '..';
+import type { GridFSBucketWriteStream } from './upload';
+
+export interface GridFSBucketReadStreamOptions {
+  sort?: number | { uploadDate: number };
+  skip?: number;
+  /** 0-based offset in bytes to start streaming from */
+  start?: number;
+  /** 0-based offset in bytes to stop streaming before */
+  end?: number;
+}
+
+export interface GridFsFile {
```

Comment:
```suggestion
export interface GridFSFile {
```
or change all the other references to be `GridFs`, but let's be consistent